examples/stupid-s.mkd

```diff
@@ -444,7 +452,7 @@ if ($_SERVER["REQUEST_METHOD"] == "GET") {
    if (empty($chid)) {
       echo $_SERVER["REMOTE_ADDR"] . ":" . $_SERVER["REMOTE_PORT"];
    } elseif ($result = mysql_query("SELECT `data` FROM `Data` " .
-                         "WHERE `chid` = '$chid'")) {
+                   "WHERE `chid` = '$chid'")) {
```

Comment:
M-x whitespace-cleanup is good, but it really messed with the code here.

Reply:
Yeah, there was only so much I was willing to fix at that point.

Reply:
Wrong direction, I'd say (the "WHERE" should be right under the "SELECT").

Reply:
Feel free to clean up, but the key point is just to see that it builds cleanly with appropriate xml2rfcs.

Reply:
Ah, there's the rub.

I think any real-world example needs to acquaint novices with the fact that this happens rarely.

But maybe there should be another "lab" example, with v3 and all...